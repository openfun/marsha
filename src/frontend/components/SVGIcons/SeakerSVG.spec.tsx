import React from 'react';
import renderer from 'react-test-renderer';
import { SpeakerSVG } from './SpeakerSVG';

it('renders SpeakerSVG correctly', () => {
  const SpeakerSVGSnapshot = renderer
    .create(<SpeakerSVG iconColor={'#035ccd'} />)
    .toJSON();
  expect(SpeakerSVGSnapshot).toMatchSnapshot();
});
