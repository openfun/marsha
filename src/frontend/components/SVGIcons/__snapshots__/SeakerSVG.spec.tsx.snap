// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SpeakerSVG correctly 1`] = `
.c0 path {
  fill: #035ccd;
}

<svg
  className="c0"
  viewBox="0 0 54 54"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <g
      transform="translate(2.5 2.5)"
    >
      <path
        d="M31.25 8.333a2.09 2.09 0 012.083 2.084v29.166a2.09 2.09 0 01-2.083 2.084h-25a2.09 2.09 0 01-2.083-2.084V10.417A2.09 2.09 0 016.25 8.333h25zm12.5 0a2.09 2.09 0 012.078 1.935l.005.149v29.166a2.09 2.09 0 01-1.935 2.078l-.148.006h-4.167a2.09 2.09 0 01-2.078-1.935l-.005-.149V10.417a2.09 2.09 0 011.935-2.078l.148-.006h4.167zm-25 19.271c-3.472 0-10.417 2.096-10.417 5.907v1.906h20.834V33.51c0-3.81-6.945-5.907-10.417-5.907zm0-13.02a5.201 5.201 0 00-5.208 5.208A5.201 5.201 0 0018.75 25a5.201 5.201 0 005.208-5.208 5.201 5.201 0 00-5.208-5.209z"
        fillRule="nonzero"
      />
    </g>
  </g>
</svg>
`;
